src/builtin/raw-arrays.scrbl

```diff
@@ -71,53 +80,86 @@ and test with @pyret-id["is=~" "testing"].
 
 @examples{
 check:
-  [raw-array: 1, 2, 3] is-not== [raw-array: 1, 2, 3]
```

Comment:
Yeah, I think I just got crossed up there since I replaced them with a raw-array-of example.